src/api/app/views/webui/cloud/upload_jobs/index.html.haml

```diff
@@ -15,7 +16,8 @@
       - @upload_jobs.each do |job|
         %tr
           %td= job.id
-          %td= job.filename
+          %td= link_to(job.filename, controller: '/webui/package', action: :binaries, project: job.project, package: job.package, repository: job.repository)
```

Comment:
> I guess that if this is already created... or the binaries are wiped out whenever a new one is created?

Only if you manually wipe the binaries. If a rebuild happens, they get wiped when the new ones get published. 
Anyway, linked to the binaries page should be fine as this will never result in a 500. Also this is valid if the file still exists or might got replaced by a rebuild.

Reply:
Done already

Reply:
I know :) Just wanted to answer to the vanished comment :smile: